packages/terra-application-navigation/tests/wdio/application-navigation-spec.js

```diff
@@ -1,392 +1,327 @@
 Terra.describeViewports('ApplicationNavigation - Large', ['large'], () => {
   describe('Extensions roll up properly', () => {
-    before(() => browser.url('/#/raw/tests/terra-application-navigation/application-navigation/extensions'));
-
     it('launch popup', () => {
+      browser.url('/raw/tests/terra-application-navigation/application-navigation/extensions');
       browser.click('[data-application-extension-rollup="true"]');
       browser.waitForVisible('[data-terra-popup-content="true"]');
       browser.pause(50);
+      Terra.validates.element('default');
     });
-
-    Terra.it.validatesElement();
   });
 
   describe('Tabs roll up properly', () => {
-    before(() => browser.url('/#/raw/tests/terra-application-navigation/application-navigation/tabs'));
-
     it('launch popup', () => {
+      browser.url('/raw/tests/terra-application-navigation/application-navigation/tabs');
       browser.waitForVisible('[data-tab-menu-inner="true"]');
       browser.click('[data-tab-menu-inner="true"]');
       browser.waitForVisible('[data-terra-popup-content="true"]');
       browser.pause(50);
+      Terra.validates.element('default', { selector: '#root' });
     });
-
-    Terra.it.validatesElement({ selector: '#root' });
   });
 
   describe('Utilities should display properly', () => {
-    before(() => browser.url('/#/raw/tests/terra-application-navigation/application-navigation/utilities'));
-
     it('launch popup', () => {
+      browser.url('/raw/tests/terra-application-navigation/application-navigation/utilities');
       browser.click('[data-application-header-utility="true"]');
       browser.waitForVisible('[data-terra-popup-content="true"]');
       browser.pause(50);
+      Terra.validates.element('default');
     });
-
-    Terra.it.validatesElement();
   });
 
   describe('Utilities should display properly when only help, settings or logout utilities are present', () => {
-    before(() => browser.url('/#/raw/tests/terra-application-navigation/application-navigation/no-custom-utility-items'));
-
+    before(() => browser.url('/raw/tests/terra-application-navigation/application-navigation/no-custom-utility-items'));
```

Comment:
Should this before hook have been deleted?

Reply:
Removed c628b6d